src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -87,6 +80,39 @@ User's MEI is available at the <i>chrome://media-engagement</i> internal page.
   </figcaption>
 </figure>
 
+### Developer switches {: #developer-switches }
+
+As a developer, you may want to change Chrome autoplay policy behaviour locally
+to test your website. The first thing you can do is disable entirely the
+autoplay policy by setting the Chrome flag "Autoplay Policy" to "No user
+gesture is required" at `chrome://flags/#autoplay-policy`.
+
+If you want a finer level of control, here are two internal media switches you
+can [play with]:
+
+- `MediaEngagementBypassAutoplayPolicies`: Toggles use of MEI to allow autoplay.
+- `PreloadMediaEngagementData`: Toggles whether sites with the highest overall MEI get autoplay by default for new users.
+
+<b>Example 1:</b> Disable whether sites with the highest overall MEI get
+autoplay by default for new users.
+```
+chrome.exe
+--disable-features=PreloadMediaEngagementData
+```
+
+<b>Example 2:</b> Disable use of MEI to allow autoplay.
+```
+chrome.exe
+--disable-features=MediaEngagementBypassAutoplayPolicies
+```
+
+<b>Example 3:</b> Disable whether sites with the highest overall MEI get
+autoplay by default for new users and use of MEI to allow autoplay.
+```
+chrome.exe
+--disable-features=PreloadMediaEngagementData,MediaEngagementBypassAutoplayPolicies
+```
```

Comment:
These examples are the kind of thing that should be in the wiki instead of in /Updates. Leave it for now. Let's discuss this off line and I'll help you get it sorted.